zenoh-kotlin/build.gradle.kts

```diff
@@ -13,7 +13,7 @@
 //
 
 group = "io.zenoh"
-version = "0.11.0-dev"
+version = project.findProperty("zkVersion") as String? ?: "0.11.0-dev"
```

Comment:
Where is this property stored?

Reply:
I'm not sure to understand, this allows to specify the version as an argument, like `gradle publishToMavenLocal -PzkVersion=0.11.0-rc`.